bellboy/actors/lead.py

```diff
@@ -57,6 +71,7 @@ def startBellboyLead(self):
 
     def stopBellboyLead(self):
         self.log.info("Stopping all child actors...")
+        self.send(self.realtime_actor, "Stopping.")
```

Comment:
These string / dict commands can be changed to msgs /Enums

Reply:
The realtime actor only takes strings anyways; I don't think the extra complexity introduced by a message-wrapping class is required at this point.